internal/validation/apiextensions/v1/composition/handler.go

```diff
@@ -50,22 +44,21 @@ const (
 	crdsIndexKey = "crd.kind.group"
 )
 
-// handler implements the admission handler for Composition.
-type handler struct {
-	reader  client.Reader
-	decoder *admission.Decoder
-	options controller.Options
-}
+// Error strings.
+const (
+	errNotComposition = "supplied object was not a Composition"
+	errUnexpectedOp   = "unexpected operation"
+	errValidationMode = "cannot get validation mode"
 
-// InjectDecoder injects the decoder.
-func (h *handler) InjectDecoder(decoder *admission.Decoder) error {
-	h.decoder = decoder
-	return nil
-}
+	errFmtTooManyCRDs = "more than one CRD found for %s.%s: %v"
+	errFmtGetCRDs     = "cannot get the needed CRDs: %v"
```

Comment:
That's good to know. In cases like this the best thing to do is to raise an issue to discuss changing the pattern. Otherwise we start to have some files that are look like @phisco wrote them according to his preferences, others like @negz wrote them etc. 🙂 Would you mind raising an issue to discuss?